src/common/options/rgw.yaml.in

```diff
@@ -2882,12 +2882,67 @@ options:
   with_legacy: true
 # extra keys that may be used for aws:kms
 # defined as map "key1=YmluCmJvb3N0CmJvb3N0LQ== key2=b3V0CnNyYwpUZXN0aW5nCg=="
+- name: rgw_crypt_s3_kms_cache_enabled
+  type: bool
+  level: advanced
+  desc: Enable SSE-KMS encryption key cache.
+  default: false
+  with_legacy: true
+  services:
+  - rgw
```

Comment:
can add `see_also:` to link related options. it probably makes sense for rgw_crypt_s3_kms_cache_enabled to link to all other new options, and have each link back to rgw_crypt_s3_kms_cache_enabled